test/tap/consoleAppender-stacktrace-test.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const test = require('tap').test;
+const sandbox = require('sandboxed-module');
+process.env.LOG4JS_USE_STACKTRACE = true;
```

Comment:
Don't set globals in the test - it'll affect other tests. Best to use sandboxed-module to pass in a process object.

Reply:
Although, see my comment below about maybe not needing to use sandboxed-module at all.